test/e2e/service.go

```diff
@@ -34,11 +34,15 @@ import (
 
 var _ = Describe("Services", func() {
 	var c *client.Client
+	// Use these in tests.  They're unique for each test to prevent name collisions.
+	var namespace0, namespace1 string
 
 	BeforeEach(func() {
 		var err error
 		c, err = loadClient()
 		Expect(err).NotTo(HaveOccurred())
+		namespace0 = "e2e-ns-" + dateStamp() + "-0"
```

Comment:
I had to shorten then name to prevent the concatenation of namespace and name from exceeding the 61 character limit imposed by GCE.  That's essentially a bug in Kubernetes, that will have to be fixed separately.  Kubernetes just concatenates them, and fils if the result exceeds the limit of the cloud provider.  I suggest that we leave as is until that bug is fixed.